src/app/home-page/new-dashboard/new-dashboard.component.html

```diff
@@ -22,34 +22,7 @@
           <span class="weight-bold">Recent Activity</span>
         </mat-card-content>
       </mat-card>
-      <mat-card class="fx-grow">
-        <div id="org-header-bar" fxLayout="row" class="mat-card-header-separate">
-          <div id="organization-number" fxLayout="row">
-            <mat-card-header fxLayout="row">
-              <mat-card-title>Organizations</mat-card-title>
-              <span class="bubble">3</span>
-            </mat-card-header>
-          </div>
-          <button
-            mat-button
-            class="private-btn small-btn-structure ml-3 mb-3"
-            id="registerOrganizationButton"
-            matTooltip="Register Organization"
-            matTooltipPosition="after"
-          >
-            <img src="../../../assets/svg/icons-plus.svg" alt="Register an Organization" class="pr-2 pb-1" />
-            Request an Organization
-          </button>
-        </div>
-        <mat-divider></mat-divider>
-        <mat-card-content class="p-3">
-          <mat-form-field appearance="outline" class="search-form-field w-100">
-            <mat-label>Search your Organizations...</mat-label>
-            <input matInput type="text" data-cy="basic-search" />
-          </mat-form-field>
-          <span class="weight-bold">Recent Activity</span>
-        </mat-card-content>
-      </mat-card>
+      <app-organization-box class="fx-grow box-container"></app-organization-box>
```

Comment:
SImilar to previous comment, I think using `fxFlex` would achieve what those two classes are doing